voice_servers/teamspeak_ARM64/egg-teamspeak3-arm64-server.json

```diff
@@ -28,6 +28,16 @@
         }
     },
     "variables": [
+        {
+            "name": "Server Query Admin Password",
+            "description": "The password for the server query admin user.",
+            "env_variable": "SERVERADMIN_PASSWORD",
+            "default_value": "changeme",
```

Comment:
this is a password what means the default value should be empty and it is normaly not required so the rules for this filed need to be `nullable|string|max:32`